pkg/controller/nodeipam/ipam/cloud_cidr_allocator_test.go

```diff
@@ -188,3 +190,112 @@ func TestNeedPodCIDRsUpdate(t *testing.T) {
 		})
 	}
 }
+
+type multiNetworkTestCase struct {
+	description            string
+	fakeNodeHandler        *testutil.FakeNodeHandler
+	northInterfaces        networkv1.NorthInterfacesAnnotation
+	additionalNodeNetworks networkv1.MultiNetworkAnnotation
+	expectedIPCapacities   map[string]int64
+}
+
+func TestUpdateMultiNetworkAnnotations(t *testing.T) {
+	testCases := []multiNetworkTestCase{
+		{
+			description: "node with no additional networks",
+			fakeNodeHandler: &testutil.FakeNodeHandler{
```

Comment:
Better to make this a common variable (or a builder function). Keeps test cases short.